Add spatial filters to the filters page (#164)
neo4j/docs-graphql: modules/ROOT/pages/types/spatial.adoc

The `Point` type is used to describe the two https://neo4j.com/docs/cypher-manual/current/values-and-types/spatial/#spatial-values-crs-geographic[Geographic coordinate reference systems] supported by Neo4j.

> In order to use it in your schema, add a field with a type `Point` to any type or types in your schema, like the following:

Reviewer: ```suggestion
In order to use it in your schema, add a field with a type `Point` to any other type(s) in your schema, like the following:
```

}
----

> The `Point` type is automatically added to your schema, in addition to the input and output types you need to query and manipulate spatial types through your API.

Reviewer: ```suggestion
The `Point` type is automatically added to your schema, in addition to the input and output types that you need to query and manipulate spatial types through your API.
```

}
----

[[cartesian-point]]
=== `CartesianPoint`

The `CartesianPoint` type is used to describe the two https://neo4j.com/docs/cypher-manual/current/values-and-types/spatial/#spatial-values-crs-cartesian[Cartesian coordinate reference systems] supported by Neo4j.

> To use it in your schema, add a field with a type `CartesianPoint` to any type or types, such as in this example:

Reviewer: ```suggestion
To use it in your schema, add a field with a type `CartesianPoint` to any type(s), such as in this example:
```

}
----

> The `CartesianPoint` type is automatically added to your schema, in addition to the input and output types you need to query and manipulate spatial types through your API.

Reviewer: ```suggestion
The `CartesianPoint` type is automatically added to your schema, in addition to the input and output types that you need to query and manipulate spatial types through your API.
```
